style.css

```diff
@@ -0,0 +1,67 @@
+/* Global */
+
+*
+{
+    box-sizing: border-box;
+    margin: 0;
+}
+
+body
+{
+    font-family: sans-serif;
+    font-size: 16px;
+    color: #000;
+    background-color: #fff;
+}
+
+.container
```

Comment:
may i know the point of this

Reply:
used in next section (top-cover) and footer